object-manipulation-1/src/get-keys.js

```diff
@@ -0,0 +1,15 @@
+/* exported getKeys */
+/*
+  ensure typeof is object
+  create empty array
+  use for... in to iterate through keys -> push to array
+  return arr
+*/
+function getKeys(obj) {
+  if (typeof obj !== 'object' || Array.isArray(obj) || obj === null) { return 'invalid'}
```

Comment:
Nice one since both `typeof` of `array` and `object` comes back as "object"..! Silly JS!